xl_auth/token/models.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+"""Token models."""
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from codecs import getencoder
+from datetime import datetime, timedelta
+from os import urandom
+
+from ..database import Column, Model, SurrogatePK, db, relationship
+
+
+class Token(SurrogatePK, Model):
+    """An OAuth2 Bearer token."""
+
+    __tablename__ = 'tokens'
+    user_id = Column(db.Integer, db.ForeignKey('users.id', ondelete='CASCADE'), nullable=False)
+    user = relationship('User')
+
+    client_id = Column(db.String(64), db.ForeignKey('clients.client_id', ondelete='CASCADE'),
+                       nullable=False)
+    client = relationship('Client')
+
+    token_type = Column(db.String(40), nullable=False)
+
+    access_token = Column(db.String(256), nullable=False, unique=True)
+    refresh_token = Column(db.String(256), nullable=False, unique=True)
+
+    expires = Column(db.DateTime, nullable=False)
+
+    scopes = Column(db.Text, nullable=False)
+
+    def __init__(self, **kwargs):
+        """Create instance."""
+        expires = datetime.now() + timedelta(seconds=3600)
```

Comment:
Instead of assigning it here, make it part of field declaration?

```python

    expires_at = Column(db.DateTime, nullable=False, 
                        default=lambda _: datetime.utcnow() + timedelta(seconds=3600))
```

Reply:
Fixed.